ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly.yaml

```diff
@@ -520,7 +520,7 @@ tests:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-aws-ipi-ovn-ipsec
 - as: aws-ipi-private-f28-disasterrecovery
-  cron: 44 8 5 * *
+  cron: 44 8 4 * *
```

Comment:
Did you manually generate the cron? Normally the cron should be generated by `generate-cron-entry.sh` tool.
```
$ ./ci-operator/config/openshift/openshift-tests-private/tools/generate-cron-entry.sh aws-ipi-private-f28-disasterrecovery openshift-openshift-tests-private-release-4.11__amd64-nightly.yaml --force
cron: 46 7 15 * *

```

Reply:
@jianlinliu I tried it in last pr, and generate-cron-entry.sh config cron with random date, it means that many dr testname be executed in same day and meanwihile there is not any dr test in some days, so it could not meet our expectation that dr test arranged almost everyday by one.

Reply:
I am okay with that, cc @liangxia

Reply:
@liangxia do you have some comments? thx

Reply:
@liangxia do you have more comments? thx

Reply:
There is an required tests failed. It blocks the PR from moving forward.

Reply:
@liangxia how to bypass this failure? thanks

Reply:
I guess you need to rebase against upstream for your branch.